tests/api/test_authenticator.py

```diff
@@ -1080,15 +845,13 @@ def test_authenticated_patron_unsupported_mechanism(self):
     def test_get_credential_from_header(self):
         integration = self._external_integration(self._str)
         basic = MockBasicAuthenticationProvider(self._default_library, integration)
-        oauth = MockOAuthAuthenticationProvider(self._default_library, "oauth1")
 
         # We can pull the password out of a Basic Auth credential
         # if a Basic Auth authentication provider is configured.
         authenticator = LibraryAuthenticator(
             _db=self._db,
             library=self._default_library,
             basic_auth_provider=basic,
-            oauth_providers=[oauth],
```

Comment:
Do we still need the bearer token signing secret on the next line if we don't have any SAML providers and we no longer have any OAuth providers?

Same below on lines 1103-1104 and possibly in other places.